addon/styles/themes/blue-sky/elements/container.overrides

```diff
@@ -24,6 +24,11 @@
     overflow: auto;
 }
 
+#example-mobile .ui.form .full.height {
```

Comment:
Instead of using id you need to add css class.
Do not forget about the template in the blueprint.